includes/admin/class-wp-job-manager-admin-notices.php

```diff
@@ -168,7 +210,8 @@ public static function display_notices() {
 		/**
 		 * Allows WPJM related plugins to set up their notice hooks.
 		 *
-		 * @since 1.32.0
+		 * @since      1.32.0
+		 * @deprecated $$next-version$$ Use the `job_manager_admin_notices` filter instead to add your own notices.
 		 */
 		do_action( 'job_manager_init_admin_notices' );
```

Comment:
We can use `do_action_deprecated` here and for the other deprecated actions. However, it looks like we aren't actually deprecating this. It is still in use above with a `add_action( 'job_manager_init_admin_notices', ... )` above, which would fire a deprecated notice on ourselves.

Reply:
That works for me. I thought I saw an issue but can't find it now. If it doesn't exist, mind creating it to track this?

Reply:
Core addon added issue: https://github.com/Automattic/WP-Job-Manager/issues/2399